content/en/blog/2023/waypoint-proxy-made-simple/index.md

```diff
@@ -0,0 +1,221 @@
+
+If we scale each namespace up to 25 deployments with 10 pods each and each waypoint deployment with 2 pods for high availability, the numbers are even more impressive - the waypoint config distribution requires just 0.8% of the configuration distribution of the sidecar, as the table below illustrates!
+
+| Config Distribution         |         Namespace 1              |       Namespace 2                |     Total     |
+| --------------------------- | -------------------------------- | -------------------------------- | ------------- |
+| Sidecars                    | 25 configurations * 250 sidecars | 25 configurations * 250 sidecars |    12500      |
+| Waypoints                   | 25 configurations * 2 waypoints  | 25 configurations * 2 waypoints  |     100       |
+| Waypoints / Sidecars        |              0.8%                |               0.8%               |      0.8%     |
+
+While we use namespace scoped waypoint proxies to illustrate the simplification above, the simplification is similar
+when you apply it to service account waypoint proxies.
+
+This reduced configuration means lower resource usage (CPU, RAM, and network bandwidth) for both the
+control plane and data plane. While users today can see similar improvements with careful usage of
+`exportTo` in your Istio networking resources or the [Sidecar](/docs/reference/config/networking/sidecar/) API,
+in ambient mode this is no longer required, making scaling a breeze.
+
+## What if my destination doesn’t have a waypoint proxy?
+
+The design of ambient mode centers around the assumption that most configuration is best implemented by the service producer, rather than the service consumer. However, this isn't always the case - sometimes we need to configure traffic management for destinations we don't control. A common example of this would be connecting to an external service with improved resilience to handle occasional connection issues (e.g. to add a timeout for calls to `example.com`).
+
+This is an area under active development in the community, where we design how traffic can be routed to your egress gateway and how you can configure the egress gateway with your desired policies. Look out for future blog posts in this area!
+
+## A deep-dive of waypoint configuration
+
+Assuming you have followed the [ambient get started guide](http://preliminary.istio.io/latest/docs/ops/ambient/getting-started/) till the [control traffic section](http://preliminary.istio.io/latest/docs/ops/ambient/getting-started/#control), you have deployed a waypoint proxy for the bookinfo-reviews service account to direct 90% traffic to reviews v1 and 10% traffic to reviews v2.
+
+Review the listeners for `reviews` waypoint proxy:
+
+{{< text bash >}}
+$ istioctl pc l deploy/bookinfo-reviews-istio-waypoint --waypoint
```

Comment:
You expanded the commands out in the later text and should here as well, proxy-config listener.